src/guides/v2.3/graphql/queries/index.md

```diff
@@ -133,6 +133,299 @@ Variables are defined separately in JSON:
+   *  Set the **Active** switch to **No**.
+   *  In the **Conditions** section, define the condition as **Category is <Subcategory_ID>**.
+   *  In the **Actions** section, set the **Apply** field to **Apply a percentage of original** and the **Discount Amount** field to **25**.
+1. Schedule an update for the catalog sales rule and assign it to the **End of Year Sale Update**. In this update, set the **Active** switch to **Yes**.
+
+#### Staging `products` query
+
+The following query returns information about a product (`24-UG05`) in the **End of Year Sale** campaign. The `Preview-Version` header contains the timestamp for a date that is within the duration of the campaign. When you include the proper headers, the query returns prices with applied discounts. Without the headers, the query returns only default prices.
+
+**Headers:**
+
+```text
+Authorization: Bearer hoyz7k697ubv5hcpq92yrtx39i7x10um
+Preview-Version: 1576389600
+```
+
+**Request:**
+
+```graphql
+{
+  products(filter: {sku: {eq: "24-UG05"}}) {
+    items {
+      name
+      sku
+      price_range {
+        minimum_price {
+          discount {
+            percent_off
+            amount_off
+          }
+          final_price {
+            value
+            currency
+          }
+          regular_price {
+            value
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+**Response with headers:**
+
+```json
+{
+  "data": {
+    "products": {
+      "items": [
+        {
+          "name": "Go-Get'r Pushup Grips",
+          "sku": "24-UG05",
+          "price_range": {
+            "minimum_price": {
+              "discount": {
+                "percent_off": 25,
+                "amount_off": 4.75
+              },
+              "final_price": {
+                "value": 14.25,
+                "currency": "USD"
+              },
+              "regular_price": {
+                "value": 19
+              }
+            }
+          }
+        }
+      ]
+    }
+  }
+}
+```
+
+**Response without headers:**
+
+```json
+{
+  "data": {
+    "products": {
+      "items": [
+        {
+          "name": "Go-Get'r Pushup Grips",
+          "sku": "24-UG05",
+          "price_range": {
+            "minimum_price": {
+              "discount": {
+                "percent_off": 0,
+                "amount_off": 0
+              },
+              "final_price": {
+                "value": 19,
+                "currency": "USD"
+              },
+              "regular_price": {
+                "value": 19
+              }
+            }
+          }
+        }
+      ]
+    }
+  }
+}
+```
+
+#### Staging `categoryList` query
+
+In this example campaign, the **End of Year Sale** subcategory and a catalog price rule are disabled when the campaign is not in effect. When you specify a valid headers, the `categoryList`query returns full details about the custom category. Otherwise, the query returns an empty array.
```

Comment:
```suggestion
In this example campaign, the **End of Year Sale** subcategory and a catalog price rule are disabled when the campaign is not in effect. When you specify valid headers, the `categoryList`query returns full details about the custom category. Otherwise, the query returns an empty array.
```